ui/src/components/PinCluster.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import L from "leaflet";
+import resting from "../assets/resting.svg";
+import { Pin, PinState } from "./Pin";
+import "react-leaflet-markercluster/dist/styles.min.css";
+import "leaflet.markercluster";
+import MarkerClusterGroup from "react-leaflet-markercluster";
+import "leaflet/dist/leaflet.css";
+import "react-leaflet-markercluster/dist/styles.min.css";
+
+const createClusterCustomIcon = function () {
+  return new L.Icon({
+    iconUrl: resting,
+    iconRetinaUrl: resting,
+  });
+};
+
+export interface PinClusterProps {
+  clusterPositions: { lat: number; lng: number }[];
+}
+
+export function PinCluster({ clusterPositions }: PinClusterProps): JSX.Element {
+  return (
+    <MarkerClusterGroup
+      showCoverageOnHover={false}
+      spiderLegPolylineOptions={{ opacity: 0 }}
+      iconCreateFunction={createClusterCustomIcon}
+    >
+      {clusterPositions.map((marker, key) => {
+        return (
+          <Pin
+            key={key}
+            position={[marker.lat, marker.lng]}
+            state={PinState.Resting}
```

Comment:
Is there a reason for the state of the pin to be a prop if we always use the same initial state?

Reply:
I copied the same format as before, plus it wouldn't always be Resting I think there'd also be redirect states in the future?

Reply:
ah okay I was just wondering, looks good to me then!